### PR TITLE
Fixup argument parsing.

### PR DESCRIPTION
**Pull request by rbtcollins**

### CODE REVIEW

**Comment by rbtcollins** [comment]
(NB: pushing up your master branch more often would help with PR's being accurate :))

File: examples/bridge.c [hunk 1]
```diff
@@ -143,7 +143,7 @@ static void
 usage(void)
 {
 	fprintf(stderr,
-	    "usage: bridge [-v] [-i ifa] [-i ifb] [-b burst] [-w wait_time] [iface]\n");
+	    "usage: bridge [-v] [-i ifa] [-i ifb] [-b burst] [-w wait_time] [ifa [ifb]]\n");
```
Comment (vmaffione) [inline examples/bridge.c]: Can you modify the last
````
[ifa [ifb [burst]]]
````
?

**Comment by rbtcollins** [comment]
Updated.

**Comment by vmaffione** [comment]
merged

**Pull request closed without merging**